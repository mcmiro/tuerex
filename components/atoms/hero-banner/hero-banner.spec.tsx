/* eslint-disable no-use-before-define */
import * as React from "react";
import {screen, render} from "@testing-library/react";
import HeroBanner from "./";

describe("Test HeroBanner component", () => {
	it("Should pass when title is defined", () => {
		render(<HeroBanner title="test" data-testid="hero-banner-test" />);

		const renderInputComponent = screen.queryByTestId("hero-banner-test");
		expect(renderInputComponent).toBeDefined();
		expect(renderInputComponent).toMatchSnapshot();
	});
});
