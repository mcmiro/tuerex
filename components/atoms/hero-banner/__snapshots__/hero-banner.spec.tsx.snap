// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test HeroBanner component Should pass when title is defined 1`] = `null`;
